### ISSUE TITLE
AttributeError: 'Client' object has no attribute 'swarm'.

### ISSUE THREAD

**Issue by pantoufleCC** [HariSekhon/Nagios-Plugins#213]
I Use "nagios-plugins" Docker image in my environment.

13 days ago, all checks are OK.
But after a rebuild on 8th of March 2019 with latest "CentOS" imkage, i've some trouble witch some docker plugins ( not all )

Error message is for exemple:
UNKNOWN: Nagios Plugin Exception: AttributeError: AttributeError: 'Client' object has no attribute 'swarm'. Please try latest version from https:/github.com/HariSekhon/nagios-plugins and if problem persists paste the full output using the --debug switch in to a ticket for a fix/update at https://github.com/HariSekhon/nagios-plugins/issues

Details for all Docker plugins:

check_docker_api_ping.py => O.K.
check_docker_images.py  => O.K.
check_docker_swarm_is_manager.py => O.K.
check_docker_swarm_service_status.py => **ERROR** AttributeError: 'function' object has no attribute 'get'
check_docker_containers.py => O.K.
check_docker_networks.py => **ERROR** AttributeError: 'function' object has no attribute 'list'.
check_docker_swarm_node_active.py => O.K.
check_docker_swarm_version.py => **ERROR** AttributeError: 'Client' object has no attribute 'swarm'
check_docker_container_status.py => **ERROR** AttributeError: 'function' object has no attribute 'get'
check_docker_swarm_enabled.py => **ERROR** AttributeError: 'Client' object has no attribute 'swarm'
check_docker_swarm_nodes.py => OK
check_docker_version.py => O.K.
check_docker_image.py => **ERROR** AttributeError: 'function' object has no attribute 'get'
check_docker_swarm_error.py => O.K.
check_docker_swarm_services.py => **ERROR**  AttributeError: 'function' object has no attribute 'list'.
check_docker_volumes.py => **ERROR**  AttributeError: 'function' object has no attribute 'list'.

**Comment by HariSekhon** [comment]
Looks like a problem with the docker library having changed.

**Comment by pantoufleCC** [comment]
Hi,
thanks for your assistance.

Is it good here that I had to create an issue for the docker image?

regards.

**Comment by HariSekhon** [comment]
Yes it's fine to create the issue here, thank you for raising this.

I've debugged the problem and it's caused by the presence of the `docker-py` python module which when present seems to supersede the `docker` module and causes this code error.

I've removed the `docker-py` package from the underlying pylib setup and rebuilt and pushed the docker images.

This should solve the issue for you. You're welcome to re-open this ticket if it's doesn't.

Thanks

Hari
